### DISCUSSION TITLE
Just a random cool idea... Bots can use Commo Rose commands?

### DISCUSSION THREAD

**Discussion by Digitalinsite** [Joe91/fun-bots#265] [Ideas]
I just had another awesome match playing with bots and thought of a cool idea. Not sure if it's possible but thought I'd throw it out there anyway. Wondering if it would be possible for bots on your team to call out random support lines like "Need Backup", "Need Ammo", "Need Medic", Go, go go!" etc etc

I reckon it would add even more personality to the bots and really make it feel like they are players in the game. 

Thanks again for the mod - absolutely epic stuff.

**Comment by Joe91** [comment]
Yeah, that idea was already on the [discord server](https://discord.com/channels/862736286774198322/1034879801539969154) as well and I really like this idea.
I was not able to get this to work yet but I hope on future VU updates to make this possible.

**Reply by Digitalinsite** [reply]
awesome stuff - super impressed at what you guys have already been able to do. So much fun playing BF3 again. 

Do you think any of the work on funbots will be able to transfer across to BF4 Warfaw Revamped when it goes mainstream?

**Reply by Firjens** [reply]
https://feedback.warsaw-revamped.com/posts/6/allow-to-play-with-or-against-ai-bots
